test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -674,14 +675,7 @@ public static boolean assertionsEnabled() {
     /**
      * Asserts that there are no files in the specified path
      */
-    public void assertPathHasBeenCleared(String path) throws Exception {
-        assertPathHasBeenCleared(PathUtils.get(path));
-    }
-
-    /**
-     * Asserts that there are no files in the specified path
-     */
-    public void assertPathHasBeenCleared(Path path) throws Exception {
+    public void assertPathHasBeenCleared(Path path) {
```

Comment:
we can maybe have an assertPathHasBeenCleared variant? then people will hopefully see both  and choose :)